collectors/nct/collector.py

```diff
@@ -20,7 +21,11 @@ def collect(conf, conn, date_from=None, date_to=None):
     base.helpers.start(conf, 'nct', {})
 
     base_url = 'https://clinicaltrials.gov/search'
-    query = {'resultsxml': True}
+    query = {
+        'resultsxml': True,
+        'rcv_s': datetime.strptime(date_from, '%Y-%m-%d').strftime('%m/%d/%Y'),
+        'rcv_e': datetime.strptime(date_to, '%Y-%m-%d').strftime('%m/%d/%Y'),
```

Comment:
This functionality was added because I missed it when I refactored this collector.